foundations/javascript_basics/object_basics.md

```diff
@@ -20,6 +20,74 @@ Objects are a *very* important part of the JavaScript language, and while for th
 1. This JavaScript.info [article on objects](https://javascript.info/object) is the best place to get started.
 1. The [MDN tutorial on objects](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics) isn't bad either, so check it out if you need another take on the subject.
 
+#### Differences between objects and primitives
+
+Earlier in the curriculum you've learned about [primitive data types](https://www.theodinproject.com/lessons/foundations-data-types-and-conditionals). Now you've seen the object data type, the main difference between the two is that primitives can contain only a single thing (string, number, etc). Objects are used to store collection of data and more complex entities.
+Besides the formal differences, there are also some technical differences, which affect how we use each data type.
+When you define a primitive variable, it will contain a copy of the information provided to it:
+
+```javascript
+const data = 42;
```

Comment:
I actually was thinking it may mislead learners for same reason, and if it's not just me then I agree it should be changed to reduce confusion, even tho it's the "correct" way to write this code which is why I decided to go for const in the end.

Reply:
Yeah, I'd say `let` is the way to go. I've used `const` in several examples before and when asked for the reasons, people have answer "it won't change because it's a const variable"